session_test.go

```diff
@@ -1868,37 +1868,3 @@ func (s *testSessionSuite) TestCastTimeToDate(c *C) {
 	date = time.Now().In(time.FixedZone("UTC", 8*int(time.Hour/time.Second)))
 	tk.MustQuery("select cast(time('12:23:34') as date)").Check(testkit.Rows(date.Format("2006-01-02")))
 }
-
-func (s *testSessionSuite) TestSetGlobalTZ(c *C) {
```

Comment:
why remove this?

Reply:
It may be deleted when I resolve the conflict.
I fix it.